thrift/shared.thrift

```diff
@@ -1384,6 +1386,7 @@ struct ListWorkflowExecutionsRequest {
   20: optional i32 pageSize
   30: optional binary nextPageToken
   40: optional string query
+  50: optional bool isCron
```

Comment:
I'm not sure if this is needed here? Will this be part of query? Or is this the only way this will be able to be requested?

The reason asked is because it will impact the way it is implemented on the frontend. i.e. users will not be able to type in the textbox as part of the advanced query. It will likely need to be an accompanying dropdown.

Reply:
I think `isCron` should be part of the `query`, and we need to change how the query parser is implemented on the server side to support this.

Reply:
Server side changes are here: https://github.com/uber/cadence/pull/4242

Since this is optional, just introducing the field shouldn't change any behavior right? Later, frontend can set this field only when it's set by the user.  Is there something else we need to be careful about?